# Hungarian translation of update-manager
# This file is distributed under the same license as the update-manager package.
# Copyright (C) 2005, Free Software Foundation, Inc.
# Gabor Kelemen <kelemeng@gnome.hu>, 2005.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager.HEAD\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2007-09-24 11:00+0200\n"
"PO-Revision-Date: 2007-09-20 17:45+0000\n"
"Last-Translator: Gabor Kelemen <kelemengabor@linuxforum.hu>\n"
"Language-Team: Hungarian <gnome@gnome.hu>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2007-09-24 08:45+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:105 ../UpdateManager/Common/utils.py:105
msgid "None"
msgstr "Nincs"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:108 ../UpdateManager/Common/utils.py:108
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:111 ../UpdateManager/Common/utils.py:111
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:114 ../UpdateManager/Common/utils.py:114
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:194 ../DistUpgrade/distro.py:401
#, python-format
msgid "Server for %s"
msgstr "Kiszolgáló a következőhöz: %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:213 ../DistUpgrade/distro.py:218
#: ../DistUpgrade/distro.py:232
msgid "Main server"
msgstr "Fő kiszolgáló"

#: ../DistUpgrade/distro.py:235
msgid "Custom servers"
msgstr "Egyéni kiszolgálók"

#: ../DistUpgrade/dist-upgrade.py:15
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"A megadott útvonalat használja frissíthető csomagokat tartalmazó CD lemez "
"kereséséhez"

#: ../DistUpgrade/dist-upgrade.py:21
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Használja a kezelőfelületet. Jelenleg elérhető: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/dist-upgrade.py:24
msgid ""
"Use special upgrade mode. Available:\n"
"desktop, server"
msgstr ""
"Használjon speciális frissítési módot. Elérhető:\n"
"asztali, kiszolgálói"

#: ../DistUpgrade/DistUpgradeCache.py:112
msgid "Broken packages"
msgstr "Törött csomagok"

#: ../DistUpgrade/DistUpgradeCache.py:113
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"A rendszere törött csomagokat tartalmaz, amelyek ezzel a szoftverrel nem "
"javíthatóak. Kérem, először javítsa ki őket a synaptic vagy az apt-get "
"segítségével."

#: ../DistUpgrade/DistUpgradeCache.py:354
msgid "Can't upgrade required meta-packages"
msgstr "A szükséges meta-csomagok nem frissíthetőek"

#: ../DistUpgrade/DistUpgradeCache.py:359
msgid "A essential package would have to be removed"
msgstr "Egy alapvető csomag eltávolításra kerülne"

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:362
#: ../UpdateManager/UpdateManager.py:850
msgid "Could not calculate the upgrade"
msgstr "Nem tudom megtervezni a frissítés menetét"

#: ../DistUpgrade/DistUpgradeCache.py:363
msgid ""
"A unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bugreport."
msgstr ""
"A frissítés megtervezése közben feloldhatatlan probléma lépett fel.\n"
"\n"
"Jelentse ezt a hibát az \"update-manager\" csomaghoz és vegye be a /var/log/"
"dist-upgrade/ könyvtárban található fájlokat a hibajelentésbe."

#. FIXME: maybe ask a question here? instead of failing?
#: ../DistUpgrade/DistUpgradeCache.py:401
msgid "Error authenticating some packages"
msgstr "Hiba történt néhány csomag hitelesítése közben"

#: ../DistUpgrade/DistUpgradeCache.py:402
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Néhány csomagot nem sikerült hitelesíteni. Lehetséges, hogy ezt átmeneti "
"hálózati probléma okozza, ezért érdemes később újra megpróbálni. Az alábbi "
"lista a hitelesíthetetlen csomagokat tartalmazza."

#: ../DistUpgrade/DistUpgradeCache.py:505
#, python-format
msgid "Can't install '%s'"
msgstr "'%s' nem telepíthető"

#: ../DistUpgrade/DistUpgradeCache.py:506
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr "Egy szükséges csomag nem telepíthető. Kérem, jelentse ezt hibaként. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:513
msgid "Can't guess meta-package"
msgstr "Nem tudom megállapítani a meta-csomagot"

#: ../DistUpgrade/DistUpgradeCache.py:514
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"A rendszere nem tartalmazza az ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop vagy edubuntu-desktop csomagok egyikét sem, és nem lehetett "
"megállapítani, hogy az Ubuntu mely változatát használja.\n"
" A folytatás előtt telepítse a fenti csomagok egyikét a synaptic vagy az apt-"
"get használatával."

#: ../DistUpgrade/DistUpgradeControler.py:92
msgid "Failed to add the CD"
msgstr "A CD hozzáadása meghiúsult"

#: ../DistUpgrade/DistUpgradeControler.py:93
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Hiba történt a CD hozzáadása során, a frissítés félbeszakad. Jelentse ezt "
"hibaként, ha ez egy érvényes Ubuntu CD.\n"
"\n"
"A hibaüzenet:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeControler.py:125
msgid "Reading cache"
msgstr "Gyorsítótár beolvasása"

#: ../DistUpgrade/DistUpgradeControler.py:182
msgid "Unable to get exclusive lock"
msgstr "A kizárolagos zárolás nem lehetséges"

#: ../DistUpgrade/DistUpgradeControler.py:183
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Ez általában azt jelenti, hogy már fut egy másik csomagkezelő alkalmazás "
"(például apt-get vagy aptitude). Kérem először zárja be azt az alkalmazást."

#: ../DistUpgrade/DistUpgradeControler.py:201
msgid "Continue running under SSH?"
msgstr "Fusson tovább SSH alatt?"

#: ../DistUpgrade/DistUpgradeControler.py:202
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, a additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Úgy tűnik, hogy a frissítés SSH alatt fut.  Nem javasolt frissítést végezni "
"SSH alatt, mert ha hiba lépne fel, nehezebb azt helyrehozni.\n"
"\n"
"Ha folytatja, egy új SSH kiszolgáló (sshd) indul a következő porton: %s.\n"
"Kívánja folytatni?"

#: ../DistUpgrade/DistUpgradeControler.py:215
msgid "Starting additional sshd"
msgstr "Új sshd indítása"

#: ../DistUpgrade/DistUpgradeControler.py:216
#, python-format
msgid ""
"To make recovery in case of failure easier a additional sshd will be started "
"on port '%s'. If anything goes wrong with the running ssh you can still "
"connect to the additional one.\n"
msgstr ""
"Ha hiba lépne fel, a helyrehozását megkönnyítendő egy új SSH kiszolgáló "
"(sshd) indul a következő porton: %s. Ha bármi probléma lépne fel az éppen "
"futó SSH-val, az újonnan indulóhoz még mindig lehet kapcsolódni.\n"

#: ../DistUpgrade/DistUpgradeControler.py:280
#: ../DistUpgrade/DistUpgradeControler.py:289
msgid "Can not upgrade"
msgstr "Nem lehet frissíteni"

#: ../DistUpgrade/DistUpgradeControler.py:281
#, python-format
msgid "A upgrade from '%s' to '%s' is not supoprted with this tool."
msgstr "Ez az eszköz nem támogatja a frissítést %s rendszerről %s rendszerre."

#: ../DistUpgrade/DistUpgradeControler.py:290
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"A Python telepítése sérült. Javítsa ki a \"/usr/bin/python\" szimbolikus "
"linket."

#: ../DistUpgrade/DistUpgradeControler.py:309
msgid "Include latest updates from the Internet?"
msgstr "Használja az internetről elérhető legújabb frissítéseket?"

#: ../DistUpgrade/DistUpgradeControler.py:310
msgid ""
"The upgrade process can automatically download the latest updates and "
"install them during the upgrade.  The upgrade will take longer, but when it "
"is complete, your system will be fully up to date.  You can choose not to do "
"this, but you should install the latest updates soon after upgrading."
msgstr ""
"A frissítési folyamat automatikusan le tudja tölteni a legújabb "
"frissítéseket és telepíteni tudja őket. A frissítés tovább fog tartani, de "
"mire elkészül, a teljes rendszere naprakész lesz. Amennyiben most nem "
"kívánja ezt megtenni, ajánlott, hogy a frissítés befejezése után minél "
"hamarabb telepítse a legújabb frissítéseket."

#: ../DistUpgrade/DistUpgradeControler.py:467
msgid "No valid mirror found"
msgstr "nem található érvényes tükör"

#: ../DistUpgrade/DistUpgradeControler.py:468
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found.This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'no' the update will cancel."
msgstr ""
"A tárolóinformációk elemzése során a program nem talált tükörbejegyzést a "
"frissítéshez. Ez akkor fordulhat elő, ha belső tükröt használ, vagy a "
"tükörinformációk elavultak.\n"
"\n"
"Mindenképpen újra kívánja írni a sources.list fájlt? Ha az \"Igen\" gombot "
"választja, akkor az összes \"%s\" bejegyzés \"%s\" bejegyzéssé lesz "
"frissítve. \n"
"A \"Nem\" kiválasztása megszakítja a frissítést."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeControler.py:485
msgid "Generate default sources?"
msgstr "Alapértelmezett források ismételt előállítása?"

#: ../DistUpgrade/DistUpgradeControler.py:486
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No' the update will "
"cancel."
msgstr ""
"A sources.list fájl elemzése után nem található érvényes bejegyzés a "
"következőhöz: %s.\n"
"\n"
"Kíván alapértelmezett bejegyzéseket adni a következőhöz: %s? Ha a Nem "
"gombott választja, a frissítés félbeszakad."

#: ../DistUpgrade/DistUpgradeControler.py:520
msgid "Repository information invalid"
msgstr "Érvénytelen csomagtároló információ"

#: ../DistUpgrade/DistUpgradeControler.py:521
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"A csomagtároló információ frissítése hibás fájlt eredményezett. Kérem, "
"jelentse ezt hibaként."

#: ../DistUpgrade/DistUpgradeControler.py:527
msgid "Third party sources disabled"
msgstr "A külső források letiltva"

#: ../DistUpgrade/DistUpgradeControler.py:528
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"A sources.list néhány harmadik féltől származó forrása le lett tiltva. A "
"frissítés után újraengedélyezheti őket a a Szoftverforrások eszközzel, vagy "
"a csomagkezelővel."

#: ../DistUpgrade/DistUpgradeControler.py:586
msgid "Error during update"
msgstr "Hiba történt a frissítés közben"

#: ../DistUpgrade/DistUpgradeControler.py:587
msgid ""
"A problem occured during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Hiba lépett fel a frissítés közben. Ez többnyire hálózati problémára utal. "
"Kérem, ellenőrizze a hálózati kapcsolatot és próbálja újra."

#: ../DistUpgrade/DistUpgradeControler.py:596
msgid "Not enough free disk space"
msgstr "Nincs elég szabad hely a merevlemezen"

#: ../DistUpgrade/DistUpgradeControler.py:597
#, python-format
msgid ""
"The upgrade aborts now. The upgrade needs a total of %s free space on disk '%"
"s'. Please free at least an additional %s of disk space on '%s'. Empty your "
"trash and remove temporary packages of former installations using 'sudo apt-"
"get clean'."
msgstr ""
"A frissítés most meg fog szakadni. Szabadítson fel legalább %s helyet a(z) %"
"s lemezen. Szabadítson fel legalább %s lemezterületet a(z) \"%s\" eszközön. "
"Ürítse a kukáját és törölje a korábbi telepítések átmeneti fájljait a \"sudo "
"apt-get clean\" parancs kiadásával."

#: ../DistUpgrade/DistUpgradeControler.py:709
msgid "Support for some applications ended"
msgstr "Egyes alkalmazások támogatása megszűnt"

#: ../DistUpgrade/DistUpgradeControler.py:710
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"If you have not enabled community maintained software (universe), these "
"packages will be suggested for removal at the end of the upgrade."
msgstr ""
"A Canonical Ltd. már nem biztosít támogatást a következő "
"szoftvercsomagokhoz. A közösségtől továbbra is kaphat támogatást. \n"
"\n"
"Ha nincsenek engedélyezve a közösség által karbantartott szoftverek "
"(universe tároló), akkor ezek a csomagok a telepítés végén eltávolításra "
"lesznek javasolva."

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeControler.py:733
msgid "Do you want to start the upgrade?"
msgstr "Biztosan el szeretné kezdeni a frissítést?"

#: ../DistUpgrade/DistUpgradeControler.py:790
#: ../DistUpgrade/DistUpgradeControler.py:840
msgid "Could not download the upgrades"
msgstr "A frissítések nem tölthetők le"

#: ../DistUpgrade/DistUpgradeControler.py:791
#: ../DistUpgrade/DistUpgradeControler.py:841
msgid ""
"The upgrade aborts now. Please check your internet connection or "
"installation media and try again. "
msgstr ""
"A frissítés most félbeszakad. Kérem, ellenőrizze a hálózati kapcsolatot vagy "
"a telepítő médiát és próbálja újra. "

#. invoke the frontend now
#: ../DistUpgrade/DistUpgradeControler.py:816
msgid ""
"The upgrade aborts now. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"A frissítés most félbeszakad. Előfordulhat, hogy a rendszer használhatatlan "
"állapotban van. Egy helyreállítás kerül futtatásra (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeControler.py:820
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bugreport.\n"
"%s"
msgstr ""
"\n"
"\n"
"A frissítés most félbeszakad. Jelentse ezt a hibát az \"update-manager\" "
"csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban található "
"fájlokat a hibajelentésbe.\n"
"%s"

#: ../DistUpgrade/DistUpgradeControler.py:824
msgid "Could not install the upgrades"
msgstr "A frissítések nem telepíthetők"

#: ../DistUpgrade/DistUpgradeControler.py:910
msgid "Remove obsolete packages?"
msgstr "Eltávolítja az elavult csomagokat?"

#: ../DistUpgrade/DistUpgradeControler.py:911
msgid "_Skip This Step"
msgstr "Ezen lé_pés kihagyása"

#: ../DistUpgrade/DistUpgradeControler.py:911
msgid "_Remove"
msgstr "_Eltávolítás"

#: ../DistUpgrade/DistUpgradeControler.py:922
msgid "Error during commit"
msgstr "Hiba a módosítások rögzítése közben"

#: ../DistUpgrade/DistUpgradeControler.py:923
msgid ""
"A problem occured during the clean-up. Please see the below message for more "
"information. "
msgstr ""
"A frissítés befejező fázisa közben hiba lépett fel. Az alábbi üzenet további "
"információkat tartalmaz a hibára vonatkozóan. "

#. generate a new cache
#: ../DistUpgrade/DistUpgradeControler.py:1037
msgid "Restoring original system state"
msgstr "A rendszer eredeti állapotának helyreállítása"

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeControler.py:1075
msgid "Required depends is not installed"
msgstr "A szükséges függőségek nincsenek telepítve"

#: ../DistUpgrade/DistUpgradeControler.py:1076
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "A következő függőség nincs telepítve: %s. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeControler.py:1203
#: ../DistUpgrade/DistUpgradeControler.py:1260
msgid "Checking package manager"
msgstr "Csomagkezelő ellenőrzése"

#: ../DistUpgrade/DistUpgradeControler.py:1208
msgid "Preparing the upgrade failed"
msgstr "A frissítés előkészítése meghiúsult"

#: ../DistUpgrade/DistUpgradeControler.py:1209
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bugreport."
msgstr ""
"A rendszer frissítésre való előkészítése meghiúsult. Jelentse ezt a hibát az "
"\"update-manager\" csomaghoz és vegye be a /var/log/dist-upgrade/ "
"könyvtárban található fájlokat a hibajelentésbe."

#: ../DistUpgrade/DistUpgradeControler.py:1224
msgid "Getting upgrade pre-requists failed"
msgstr "A frissítés előfeltételeinek lekérése meghiúsult"

#: ../DistUpgrade/DistUpgradeControler.py:1225
msgid ""
"The system was unable to get the pre-requists for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bugreport."
msgstr ""
"A rendszer nem volt képes lekérni a frissítés előfeltételeit. A frissítés "
"most megszakad és helyreáll a rendszer eredeti állapota.\n"
"\n"
"Jelentse ezt a hibát az \"update-manager\" csomaghoz és vegye be a /var/log/"
"dist-upgrade/ könyvtárban található fájlokat a hibajelentésbe."

#: ../DistUpgrade/DistUpgradeControler.py:1246
msgid "Updating repository information"
msgstr "Csomagtároló információ frissítése"

#: ../DistUpgrade/DistUpgradeControler.py:1271
msgid "Invalid package information"
msgstr "Érvénytelen csomaginformációk"

#: ../DistUpgrade/DistUpgradeControler.py:1272
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the "
"bugreport."
msgstr ""
"A csomaginformációk frissítése után a(z) \"%s\" alapvető csomag nem "
"található többé.\n"
"Ez egy komoly problémát jelez, jelentse ezt a hibát az \"update-manager\" "
"csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban található "
"fájlokat a hibajelentésbe."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeControler.py:1283
msgid "Asking for confirmation"
msgstr "Megerősítés kérése"

#: ../DistUpgrade/DistUpgradeControler.py:1289 ../update-manager:83
msgid "Fetching"
msgstr "Letöltés"

#: ../DistUpgrade/DistUpgradeControler.py:1299 ../update-manager:87
msgid "Upgrading"
msgstr "Frissítés folyamatban"

#: ../DistUpgrade/DistUpgradeControler.py:1306
msgid "Searching for obsolete software"
msgstr "Elavult szoftverek keresése"

#: ../DistUpgrade/DistUpgradeControler.py:1311
msgid "System upgrade is complete."
msgstr "A rendszer frissítése befejeződött."

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:107
#: ../DistUpgrade/DistUpgradeViewKDE.py:115
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Helyezze be a következő médiát: '%s', ebbe a meghajtóba: '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:126
#: ../DistUpgrade/DistUpgradeViewKDE.py:128
msgid "Fetching is complete"
msgstr "A letöltés befejeződött"

#: ../DistUpgrade/DistUpgradeViewGtk.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:141
#, python-format
msgid "Fetching file %li of %li at %sb/s"
msgstr "Fájl letöltése (%li., összesen: %li), sebesség: %sB/mp"

#: ../DistUpgrade/DistUpgradeViewGtk.py:139
#: ../DistUpgrade/DistUpgradeViewGtk.py:270
#: ../DistUpgrade/DistUpgradeViewKDE.py:142
#: ../DistUpgrade/DistUpgradeViewKDE.py:271
#, python-format
msgid "About %s remaining"
msgstr "Kb. %s van hátra"

#: ../DistUpgrade/DistUpgradeViewGtk.py:141
#: ../DistUpgrade/DistUpgradeViewKDE.py:144
#, python-format
msgid "Fetching file %li of %li"
msgstr "Fájl letöltése: %li/%li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:174
#: ../DistUpgrade/DistUpgradeViewKDE.py:173
msgid "Applying changes"
msgstr "Módosítások alkalmazása..."

#: ../DistUpgrade/DistUpgradeViewGtk.py:199
#: ../DistUpgrade/DistUpgradeViewKDE.py:184
#, python-format
msgid "Could not install '%s'"
msgstr "'%s' nem telepíthető"

#: ../DistUpgrade/DistUpgradeViewGtk.py:200
#: ../DistUpgrade/DistUpgradeViewKDE.py:185
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may be in a not working "
"state. Please consider submitting a bugreport about it."
msgstr ""
"A frissítés folytatódik, de a(z) %s lehet, hogy nem fog működni. Ebben az "
"esetben kérjük, jelentse be a hibát."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Lecseréli a személyre szabott\n"
"\"%s\"\n"
"konfigurációs fájlt?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:204
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"A beállítófájl összes módosítása elvész, ha lecseréli az újabb verzióra."

#: ../DistUpgrade/DistUpgradeViewGtk.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
msgid "The 'diff' command was not found"
msgstr "A 'diff' parancs nem található"

#: ../DistUpgrade/DistUpgradeViewGtk.py:407
#: ../DistUpgrade/DistUpgradeViewText.py:77
msgid "A fatal error occured"
msgstr "Végzetes hiba történt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:408
msgid ""
"Please report this as a bug (if you haven't already) and include thefiles /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade aborts now.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Jelentse ezt hibaként (ha még nem tette) és a hibajelentéshez mellékelje a /"
"var/log/dist-upgrade/main.log és /var/log/dist-upgrade/apt.log fájlokat. A "
"frissítés most félbeszakad.\n"
"Az eredeti sources.list /etc/apt/sources.list.distUpgrade néven került "
"mentésre."

#. FIXME: make those two seperate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeViewGtk.py:542
#: ../DistUpgrade/DistUpgradeViewKDE.py:576
#: ../DistUpgrade/DistUpgradeViewText.py:128
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d csomag el lesz távolítva."

#: ../DistUpgrade/DistUpgradeViewGtk.py:547
#: ../DistUpgrade/DistUpgradeViewKDE.py:581
#: ../DistUpgrade/DistUpgradeViewText.py:133
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d új csomag kerül telepítésre."

#: ../DistUpgrade/DistUpgradeViewGtk.py:553
#: ../DistUpgrade/DistUpgradeViewKDE.py:587
#: ../DistUpgrade/DistUpgradeViewText.py:139
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d csomag frissítve lesz."

#: ../DistUpgrade/DistUpgradeViewGtk.py:558
#: ../DistUpgrade/DistUpgradeViewText.py:144
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Összes letöltendő adatmennyiség: %s. "

#: ../DistUpgrade/DistUpgradeViewGtk.py:564
#: ../DistUpgrade/DistUpgradeViewKDE.py:607
#: ../DistUpgrade/DistUpgradeViewKDE.py:615
#: ../DistUpgrade/DistUpgradeViewText.py:149
msgid ""
"Fetching and installing the upgrade can take several hours and cannot be "
"canceled at any time later."
msgstr ""
"A frissítés letöltése és telepítése több órát is igénybe vehet és a "
"későbbiek során nem lehet bármikor megszakítani."

#: ../DistUpgrade/DistUpgradeViewGtk.py:567
#: ../DistUpgrade/DistUpgradeViewKDE.py:610
#: ../DistUpgrade/DistUpgradeViewKDE.py:618
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Az esetleges adatvesztés elkerülése érdekében zárjon be minden nyitott "
"alkalmazást és dokumentumot."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeViewGtk.py:573
#: ../DistUpgrade/DistUpgradeViewKDE.py:624
#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../UpdateManager/UpdateManager.py:658
msgid "Your system is up-to-date"
msgstr "A rendszere naprakész"

#: ../DistUpgrade/DistUpgradeViewGtk.py:574
#: ../DistUpgrade/DistUpgradeViewKDE.py:625
#: ../DistUpgrade/DistUpgradeViewText.py:156
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Nem állnak rendelkezésre frissítések a rendszeréhez. A frissítés most "
"megszakad."

#: ../DistUpgrade/DistUpgradeViewGtk.py:590
#, python-format
msgid "<b>Remove %s</b>"
msgstr "<b>%s eltávolítása</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:592
#: ../DistUpgrade/DistUpgradeViewKDE.py:648
#, python-format
msgid "Install %s"
msgstr "%s telepítése"

#: ../DistUpgrade/DistUpgradeViewGtk.py:594
#: ../DistUpgrade/DistUpgradeViewKDE.py:650
#, python-format
msgid "Upgrade %s"
msgstr "%s frissítése"

#: ../DistUpgrade/DistUpgradeViewKDE.py:116
msgid "Media Change"
msgstr "Adathordozó-csere"

#: ../DistUpgrade/DistUpgradeViewKDE.py:231
msgid "Show Difference >>>"
msgstr "Különbség megjelenítése >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:234
msgid "<<< Hide Difference"
msgstr "<<< Különbség elrejtése"

#: ../DistUpgrade/DistUpgradeViewKDE.py:473
msgid "Show Terminal >>>"
msgstr "Terminál megjelenítése >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:476
msgid "<<< Hide Terminal"
msgstr "<<< Terminál elrejtése"

#. string = _("<p>You have to download a total of %s. ")
#. msg += string % apt_pkg.SizeToStr(downloadSize)
#. msg += _("<p>You have to download a total of %s. ") %#: ../DistUpgrade/DistUpgradeViewKDE.py:593
#: ../DistUpgrade/DistUpgradeViewKDE.py:599
#, python-format
msgid "<p>You have to download a total of %s. "
msgstr "<p>Összes letöltendő adatmennyiség: %s. "

#: ../DistUpgrade/DistUpgradeViewKDE.py:646
#, python-format
msgid "Remove %s"
msgstr "%s eltávolítása"

#: ../DistUpgrade/DistUpgradeViewKDE.py:666
#: ../DistUpgrade/DistUpgrade.glade.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Megszakítja a folyamatban lévő frissítést?</big></b>\n"
"\n"
"A rendszer használhatatlan állapotban maradhat, ha megszakítja a frissítést. "
"Erősen javasoljuk, hogy folytassa a frissítést."

#: ../DistUpgrade/DistUpgradeViewKDE.py:670
msgid "Cancel Upgrade?"
msgstr "Megszakítja a frissítést?"

#: ../DistUpgrade/DistUpgradeView.py:33
#, python-format
msgid "%li days %li hours %li minutes"
msgstr "%li nap, %li óra és %li perc"

#: ../DistUpgrade/DistUpgradeView.py:35
#, python-format
msgid "%li hours %li minutes"
msgstr "%li óra és %li perc"

#: ../DistUpgrade/DistUpgradeView.py:37
#, python-format
msgid "%li minutes"
msgstr "%li perc"

#: ../DistUpgrade/DistUpgradeView.py:38
#, python-format
msgid "%li seconds"
msgstr "%li másodperc"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:44
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem"
msgstr ""
"A letöltés körülbelül %sig tart 1 Mbit DSL kapcsolaton és %sig 56k modem "
"használatával"

#: ../DistUpgrade/DistUpgradeView.py:150
msgid "Reboot required"
msgstr "Újraindítás szükséges"

#: ../DistUpgrade/DistUpgradeView.py:151
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"A frissítés elkészült, de a befejezéshez újra kell indítani a rendszert. "
"Újraindítja most?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:123
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:123
msgid "Could not run the upgrade tool"
msgstr "Nem sikerült futtatni a frissítőeszközt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:124
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:124
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Feltehetőleg ez egy hiba a frissítőeszközben. Kérem, jelentse ezt hibaként."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:150
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:150
msgid "Upgrade tool signature"
msgstr "Frissítőeszköz aláírása"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:153
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:153
msgid "Upgrade tool"
msgstr "Frissítőeszköz"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:177
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:177
msgid "Failed to fetch"
msgstr "A letöltés meghiúsult"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:178
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:178
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"A frissítés letöltése meghiúsult. Lehetséges, hogy hálózati probléma áll "
"fenn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:182
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:182
msgid "Failed to extract"
msgstr "A kibontás meghiúsult"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:183
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:183
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"A frissítés kibontása meghiúsult. Probléma lehet a hálózattal vagy a "
"kiszolgálóval. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:188
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:188
msgid "Verfication failed"
msgstr "Az ellenőrzés meghiúsult"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:189
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:189
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"A frissítés ellenőrzése meghiúsult. Probléma lehet a hálózattal vagy a "
"kiszolgálóval. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:194
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:194
msgid "Authentication failed"
msgstr "Hitelesítés sikertelen"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:195
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:195
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"A frissítés hitelesítése meghiúsult. Probléma lehet a hálózattal vagy a "
"kiszolgálóval. "

#: ../DistUpgrade/DistUpgradeViewText.py:78
#, fuzzy
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade aborts "
"now.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Jelentse ezt hibaként (ha még nem tette) és a hibajelentéshez mellékelje a /"
"var/log/dist-upgrade/main.log és /var/log/dist-upgrade/apt.log fájlokat. A "
"frissítés most félbeszakad.\n"
"Az eredeti sources.list /etc/apt/sources.list.distUpgrade néven került "
"mentésre."

#: ../DistUpgrade/DistUpgradeViewText.py:101
msgid "Aborting"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:164
#: ../DistUpgrade/DistUpgradeViewText.py:184
#, fuzzy
msgid "Continue [yN] "
msgstr "Folytatás"

#: ../DistUpgrade/DistUpgradeViewText.py:164
#, fuzzy
msgid "Details [d]"
msgstr "Részletek"

#. TRANSLATORS: the "y" is "yes"
#: ../DistUpgrade/DistUpgradeViewText.py:168
#: ../DistUpgrade/DistUpgradeViewText.py:186
msgid "y"
msgstr ""

#. TRANSLATORS: the "n" is "no"
#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "n"
msgstr ""

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:174
msgid "d"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:176
#, fuzzy, python-format
msgid "Remove: %s\n"
msgstr "%s eltávolítása"

#: ../DistUpgrade/DistUpgradeViewText.py:177
#, fuzzy, python-format
msgid "Install: %s\n"
msgstr "%s telepítése"

#: ../DistUpgrade/DistUpgradeViewText.py:178
#, fuzzy, python-format
msgid "Upgrade: %s\n"
msgstr "%s frissítése"

#: ../DistUpgrade/DistUpgradeViewText.py:191
#, fuzzy
msgid "Restart required"
msgstr "Újraindítás szükséges"

#: ../DistUpgrade/DistUpgradeViewText.py:192
msgid "To fully ugprade, please restart"
msgstr ""

#: ../DistUpgrade/DistUpgrade.glade.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.glade.h:5
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Indítsa újra a rendszert a frissítés befejezéséhez</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:6
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Megkezdi a frissítést?</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:7
msgid "<b><big>Upgrading Ubuntu to version 7.10</big></b>"
msgstr "<b><big>Az Ubuntu frissítése a 7.10-es változatra.</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:8
msgid "Cleaning up"
msgstr "Frissítés befejezése"

#: ../DistUpgrade/DistUpgrade.glade.h:9
msgid "Details"
msgstr "Részletek"

#: ../DistUpgrade/DistUpgrade.glade.h:10
msgid "Difference between the files"
msgstr "Fájlok közti különbség"

#: ../DistUpgrade/DistUpgrade.glade.h:11
msgid "Distribution Upgrade"
msgstr "Disztribúciófrissítés"

#: ../DistUpgrade/DistUpgrade.glade.h:12
msgid "Fetching the upgrades"
msgstr "A frissítések letöltése"

#: ../DistUpgrade/DistUpgrade.glade.h:13
msgid "Installing the upgrades"
msgstr "A frissítések telepítése"

#: ../DistUpgrade/DistUpgrade.glade.h:14
msgid "Modifying the software channels"
msgstr "Szoftvercsatornák módosítása"

#: ../DistUpgrade/DistUpgrade.glade.h:15
msgid "Preparing the upgrade"
msgstr "Frissítés előkészítése"

#: ../DistUpgrade/DistUpgrade.glade.h:16
msgid "Restarting the system"
msgstr "A rendszer újraindítása"

#: ../DistUpgrade/DistUpgrade.glade.h:17
msgid "Terminal"
msgstr "Terminál"

#: ../DistUpgrade/DistUpgrade.glade.h:18
msgid "_Cancel Upgrade"
msgstr "Frissítés _megszakítása"

#: ../DistUpgrade/DistUpgrade.glade.h:19
msgid "_Continue"
msgstr "Folytatás"

#: ../DistUpgrade/DistUpgrade.glade.h:20
msgid "_Keep"
msgstr "_Megtartás"

#: ../DistUpgrade/DistUpgrade.glade.h:21
msgid "_Replace"
msgstr "_Csere"

#: ../DistUpgrade/DistUpgrade.glade.h:22
msgid "_Report Bug"
msgstr "_Hibabejelentés"

#: ../DistUpgrade/DistUpgrade.glade.h:23
msgid "_Restart Now"
msgstr "Újrain_dítás most"

#: ../DistUpgrade/DistUpgrade.glade.h:24
msgid "_Resume Upgrade"
msgstr "Frissítés _folytatása"

#: ../DistUpgrade/DistUpgrade.glade.h:25
msgid "_Start Upgrade"
msgstr "Frissítés _indítása"

#: ../UpdateManager/DistUpgradeFetcher.py:79
msgid "Could not find the release notes"
msgstr "Nem érhetők el a kiadási megjegyzések"

#: ../UpdateManager/DistUpgradeFetcher.py:80
msgid "The server may be overloaded. "
msgstr "A kiszolgáló túl lehet terhelve. "

#: ../UpdateManager/DistUpgradeFetcher.py:90
msgid "Could not download the release notes"
msgstr "A kiadási megjegyzések nem tölthetők le"

#: ../UpdateManager/DistUpgradeFetcher.py:91
msgid "Please check your internet connection."
msgstr "Kérjük ellenőrizze az internetkapcsolatát."

#: ../UpdateManager/GtkProgress.py:108
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"%(current)li. fájl letöltése, összesen: %(total)li, sebesség: %(speed)s/mp"

#: ../UpdateManager/GtkProgress.py:113
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "%(current)li. fájl letöltése, összesen: %(total)li"

#: ../UpdateManager/UpdateManager.py:235
msgid "The list of changes is not available"
msgstr "A módosítások listája nem érhető el"

#: ../UpdateManager/UpdateManager.py:241
msgid ""
"The list of changes is not available yet.\n"
"Please try again later."
msgstr ""
"A módosítások listája még nem érhető el.\n"
"Próbálkozzon később."

#: ../UpdateManager/UpdateManager.py:246
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"A módosítások listájának letöltése meghiúsult.\n"
"Ellenőrizze az internetkapcsolatát."

#: ../UpdateManager/UpdateManager.py:273
msgid "Important security updates"
msgstr "Fontos biztonsági frissítések"

#: ../UpdateManager/UpdateManager.py:274
msgid "Recommended updates"
msgstr "Ajánlott frissítések"

#: ../UpdateManager/UpdateManager.py:275
msgid "Proposed updates"
msgstr "Javasolt frissítések"

#: ../UpdateManager/UpdateManager.py:276
msgid "Backports"
msgstr "Visszaportolt csomagok"

#: ../UpdateManager/UpdateManager.py:277
msgid "Distribution updates"
msgstr "Disztribúciófrissítések"

#: ../UpdateManager/UpdateManager.py:282
msgid "Other updates"
msgstr "Egyéb frissítések"

#: ../UpdateManager/UpdateManager.py:508
#, python-format
msgid "Version %s: \n"
msgstr "%s verzió: \n"

#: ../UpdateManager/UpdateManager.py:569
msgid "Downloading list of changes..."
msgstr "Változások listájának letöltése..."

#: ../UpdateManager/UpdateManager.py:596
msgid "_Uncheck All"
msgstr "_Kijelölések törlése"

#: ../UpdateManager/UpdateManager.py:602
msgid "_Check All"
msgstr "Összes _ellenőrzése"

#. TRANSLATORS: b stands for Bytes
#: ../UpdateManager/UpdateManager.py:646 ../UpdateManager/UpdateManager.py:673
#, python-format
msgid "Download size: %s"
msgstr "Letöltés mérete: %s"

#: ../UpdateManager/UpdateManager.py:650
msgid "Unknown download size"
msgstr "A letöltés mérete ismeretlen"

#: ../UpdateManager/UpdateManager.py:669
#, python-format
msgid "You can install %s update"
msgid_plural "You can install %s updates"
msgstr[0] "%s frissítést telepíthet"

#: ../UpdateManager/UpdateManager.py:702
msgid "Please wait, this can take some time."
msgstr "Kis türelmet, ez eltarthat egy ideig."

#: ../UpdateManager/UpdateManager.py:704
msgid "Update is complete"
msgstr "A frissítés befejeződött"

#: ../UpdateManager/UpdateManager.py:756
msgid "Checking for updates"
msgstr "Frissítések keresése"

#: ../UpdateManager/UpdateManager.py:828
msgid "Could not initialize the package information"
msgstr "A csomaginformációk inicializálása meghiúsult"

#: ../UpdateManager/UpdateManager.py:829
msgid ""
"A unresolvable problem occurred while initializing the package information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"A csomaginformációk inicializálása közben feloldhatatlan probléma lépett "
"fel.\n"
"\n"
"Jelentse ezt a hibát az \"update-manager\" csomaghoz és vegye be a következő "
"üzenetet a hibajelentésbe:\n"

#: ../UpdateManager/UpdateManager.py:851
msgid ""
"A unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"A frissítés kiszámítása közben feloldhatatlan probléma lépett fel.\n"
"\n"
"Jelentse ezt a hibát az \"update-manager\" csomaghoz és vegye be a következő "
"üzenetet a hibajelentésbe:"

#: ../UpdateManager/UpdateManager.py:876
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Régi verzió: %(old_version)s, új verzió:  %(new_version)s"

#: ../UpdateManager/UpdateManager.py:880
#, python-format
msgid "Version %s"
msgstr "%s verzió"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:882
#, python-format
msgid "(Size: %s)"
msgstr "(Méret: %s)"

#: ../UpdateManager/UpdateManager.py:893
msgid "Your distribution is not supported anymore"
msgstr "A terjesztés már nem támogatott"

#: ../UpdateManager/UpdateManager.py:894
msgid ""
"You will not get any further security fixes or critical updates. Upgrade to "
"a later version of Ubuntu Linux. See http://www.ubuntu.com for more "
"information on upgrading."
msgstr ""
"Nem fog további biztonsági javításokat vagy kritikus frissítéseket kapni. "
"Frissítsen az Ubuntu Linux egy újabb változatára. A frissítéssel kapcsolatos "
"információkat az http://www.ubuntu.com weboldalon talál."

#: ../UpdateManager/UpdateManager.py:909
msgid "Downloading the upgrade tool"
msgstr "Frissítőeszköz letöltése"

#: ../UpdateManager/UpdateManager.py:911
msgid "The upgrade tool will guide you through the upgrade process."
msgstr "A frissítőeszköz végigvezeti Önt a frissítési folyamaton."

#: ../UpdateManager/UpdateManager.py:919
#, python-format
msgid "<b>New distribution release '%s' is available</b>"
msgstr "<b>A disztribúció új \"%s\" kiadása elérhető</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:958
msgid "Software index is broken"
msgstr "A szoftverindex sérült"

#: ../UpdateManager/UpdateManager.py:959
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Lehetetlen szoftvereket telepíteni vagy törölni. Használja a Synaptic "
"csomagkezelőt vagy futtassa a \"sudo apt-get install -f\" parancsot egy "
"terminálban a probléma megoldása érdekében."

#: ../data/glade/UpdateManager.glade.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Internet Updates</i> tab."
msgstr ""
"<b><big>Saját kezűleg kell megkeresnie a rendelkezésre álló frissítéseket</"
"big></b>\n"
"\n"
"A rendszere jelenleg nem keresi automatikusan a frissítéseket. Ezt a "
"viselkedést az <i>Internetes frissítések</i> lap <i>Szoftverforrások</i> "
"szakaszában változtathatja meg."

#: ../data/glade/UpdateManager.glade.h:4
msgid "<big><b>Keep your system up-to-date</b></big>"
msgstr "<big><b>Tartsa naprakészen a rendszerét</b></big>"

#: ../data/glade/UpdateManager.glade.h:5
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Nem minden frissítés telepíthető</b></big>"

#: ../data/glade/UpdateManager.glade.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Frissítéskezelő indítása</b></big>"

#: ../data/glade/UpdateManager.glade.h:7
msgid "Changes"
msgstr "Módosítások"

#: ../data/glade/UpdateManager.glade.h:8
msgid "Chec_k"
msgstr "_Ellenőrzés"

#: ../data/glade/UpdateManager.glade.h:9
msgid "Check the software channels for new updates"
msgstr "Frissítések keresése a szoftvercsatornákon"

#: ../data/glade/UpdateManager.glade.h:10
msgid "Description"
msgstr "Leírás"

#: ../data/glade/UpdateManager.glade.h:11
msgid "Description of update"
msgstr "Frissítés leírása"

#: ../data/glade/UpdateManager.glade.h:12
msgid "Release Notes"
msgstr "Kiadási megjegyzések"

#: ../data/glade/UpdateManager.glade.h:13
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" ◦ A previous upgrade which didn't complete\n"
" ◦ Unofficial software packages not provided by Ubuntu\n"
" ◦ Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Részleges frissítés futtatása, a lehető legtöbb frissítés telepítése "
"érdekében. \n"
"\n"
"Ezt egy:\n"
" ◦ korábbi befejezetlen frissítés\n"
" ◦ nem hivatalos, nem az Ubuntu által szállított szoftvercsomagok\n"
" ◦ az Ubuntu fejlesztői verziójának normális változásai okozhatják"

#: ../data/glade/UpdateManager.glade.h:19
msgid "Show progress of single files"
msgstr "Egyes fájlok állapotának mutatása"

#: ../data/glade/UpdateManager.glade.h:20
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"A szoftverfrissítések kijavítják a programhibákat és biztonsági "
"sebezhetőségeket, illetve új szolgáltatásokat biztosítanak."

#: ../data/glade/UpdateManager.glade.h:21
msgid "U_pgrade"
msgstr "_Frissítés"

#: ../data/glade/UpdateManager.glade.h:22
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Frissítéskezelő"

#: ../data/glade/UpdateManager.glade.h:23
msgid "Upgrade to the latest version of Ubuntu"
msgstr "Frissítés az Ubuntu legújabb változatára"

#: ../data/glade/UpdateManager.glade.h:24
msgid "_Check"
msgstr "_Ellenőrzés"

#: ../data/glade/UpdateManager.glade.h:25
msgid "_Hide this information in the future"
msgstr "_A jövőben ne mutassa ezt az információt"

#: ../data/glade/UpdateManager.glade.h:26
msgid "_Install Updates"
msgstr "_Telepítés"

#: ../data/glade/UpdateManager.glade.h:27
msgid "_Partial Upgrade"
msgstr "_Részleges frissítés"

#: ../data/glade/UpdateManager.glade.h:28
msgid "_Upgrade"
msgstr "_Frissítés"

#: ../data/glade/UpdateManager.glade.h:29
msgid "updates"
msgstr "frissítés"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Rendelkezésre álló frissítések megjelenítése és telepítése"

#: ../data/update-manager.desktop.in.h:2
msgid "System Update Tool"
msgstr "Rendszerfrissítő eszköz"

#: ../data/update-manager.schemas.in.h:1
msgid ""
"Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"Az aktuális terjesztés új verziójának automatikus keresése és a frissítés "
"felajánlása (ha lehetséges)."

#: ../data/update-manager.schemas.in.h:2
msgid "Check for new distribution releases"
msgstr "Új disztribúciókiadások keresése"

#: ../data/update-manager.schemas.in.h:3
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"Ha a frissítések automatikus keresése le van tiltva, akkor a csatornalistát "
"kézzel kell újratölteni. Ezzel a beállítással elrejtheti az ilyen "
"helyzetekben megjelenő emlékeztetőt."

#: ../data/update-manager.schemas.in.h:4
msgid "Remind to reload the channel list"
msgstr "Figyelmeztessen, ha újra kell tölteni a csatornalistát"

#: ../data/update-manager.schemas.in.h:5
msgid "Show details of an update"
msgstr "Frissítés részleteinek megjelenítése"

#: ../data/update-manager.schemas.in.h:6
msgid "Stores the size of the update-manager dialog"
msgstr "A frissítéskezelő ablak méretének tárolása"

#: ../data/update-manager.schemas.in.h:7
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""
"Eltárolja a módosítások listáját és a leírást tartalmazó kiterjesztő "
"állapotát"

#: ../data/update-manager.schemas.in.h:8
msgid "The window size"
msgstr "Az ablak mérete"

#: ../update-manager:51
msgid "Check if a new distribution release is available"
msgstr "Új disztribúciókiadások keresése"

#: ../update-manager:54 ../do-release-upgrade:19
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Ellenőrzi, hogy lehetséges-e frissíteni a legújabb fejlesztői kiadásra"

#: ../update-manager:58 ../update-manager:61
msgid "Try to run a dist-upgrade"
msgstr "Megpróbál disztribúciófrissítést futtatni"

#: ../update-manager:74
msgid "Running partial upgrade"
msgstr "Részleges frissítés futtatása"

#: ../update-manager:92
msgid "Upgrade complete"
msgstr "A frissítés befejeződött"

#: ../update-manager:93
msgid "The upgrade was completed."
msgstr "A rendszer frissítése befejeződött."

#: ../do-release-upgrade:23
msgid "Run the specified frontend"
msgstr "A megadott előtét futtatása"

#: ../do-release-upgrade:26
msgid "Run in a special upgrade mode"
msgstr "Futtatás speciális frissítő módban"

#: ../do-release-upgrade:30
msgid "Checking for a new ubuntu release"
msgstr "Új Ubuntu kiadás keresése"

#: ../do-release-upgrade:36
msgid "No new release found"
msgstr "Nem található új kiadás"

#~ msgid "Fetching backport of '%s'"
#~ msgstr "\"%s\" visszaportolt változatának letöltése"

#~ msgid ""
#~ "The upgrade aborts now. Please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the "
#~ "bugreport."
#~ msgstr ""
#~ "A frissítés most félbeszakad. Jelentse ezt a hibát az \"update-manager\" "
#~ "csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban található "
#~ "fájlokat a hibajelentésbe."

#, fuzzy
#~ msgid "Fetching file %li of %li at %s/s"
#~ msgstr "Fájl letöltése (%li., összesen: %li), sebesség: %s/mp"

#~ msgid "Fetch data from the network for the upgrade?"
#~ msgstr "Letölt adatokat a hálózatról a frissítéshez?"

#~ msgid ""
#~ "The upgrade can use the network to check the latest updates and to fetch "
#~ "packages that are not on the current CD.\n"
#~ "If you have fast or inexpensive network access you should answer 'Yes' "
#~ "here. If networking is expensive for you choose 'No'."
#~ msgstr ""
#~ "A frissítés használhatja a hálózatot a legújabb frissítések ellenőrzésére "
#~ "és letöltheti a jelenlegi CD-n el nem érhető csomagokat.\n"
#~ "Ha gyors vagy olcsó hálózati kapcsolattal rendelkezik, válaszoljon "
#~ "igennel. Ha a hálózat elérése drásga, válaszoljon nemmel."

#~ msgid "Changes and description of the update"
#~ msgstr "A frissítés módosításai és leírása"

#~ msgid "Software Updates"
#~ msgstr "Szoftverfrissítések"

#~ msgid "changes"
#~ msgstr "módosítás"

#~ msgid "Daily"
#~ msgstr "Naponta"

#~ msgid "Every two days"
#~ msgstr "Kétnaponta"

#~ msgid "Weekly"
#~ msgstr "Hetente"

#~ msgid "Every two weeks"
#~ msgstr "Minden két hétben"

#~ msgid "Every %s days"
#~ msgstr "%s naponta"

#~ msgid "After one week"
#~ msgstr "Egy hét után"

#~ msgid "After two weeks"
#~ msgstr "Két hét után"

#~ msgid "After one month"
#~ msgstr "Egy hónap után"

#~ msgid "After %s days"
#~ msgstr "%s nap után"

#~ msgid "%s updates"
#~ msgstr "%s frissítések"

#~ msgid "%s (%s)"
#~ msgstr "%s (%s)"

#~ msgid "Nearest server"
#~ msgstr "Legközelebbi kiszolgáló"

#~ msgid "Software Channel"
#~ msgstr "Szoftvercsatorna"

#~ msgid "Active"
#~ msgstr "Aktív"

#~ msgid "(Source Code)"
#~ msgstr "(Forráskód)"

#~ msgid "Source Code"
#~ msgstr "Forráskód"

#~ msgid "Import key"
#~ msgstr "Kulcs importálása"

#~ msgid "Error importing selected file"
#~ msgstr "Hiba a kiválasztott fájl importálása közben"

#~ msgid "The selected file may not be a GPG key file or it might be corrupt."
#~ msgstr "A kiválasztott fájl vagy nem GPG kulcsfájl, vagy sérült."

#~ msgid "Error removing the key"
#~ msgstr "Hiba a kulcs eltávolítása közben"

#~ msgid ""
#~ "The key you selected could not be removed. Please report this as a bug."
#~ msgstr ""
#~ "Az Ön által kijelölt kulcs nem távolítható el. Kérem jelentse ezt "
#~ "hibaként."

#, fuzzy
#~ msgid ""
#~ "<big><b>Error scanning the CD</b></big>\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "<big><b>Hiba történt a CD beolvasása közben</b></big>\n"
#~ "\n"
#~ "%s"

#~ msgid "Please enter a name for the disc"
#~ msgstr "Kérem, adja meg a lemez nevét"

#~ msgid "Please insert a disc in the drive:"
#~ msgstr "Kérem, helyezzen be egy lemezt a meghajtóba:"

#~ msgid "<b>Automatic updates</b>"
#~ msgstr "<b>Automatikus frissítések</b>"

#~ msgid "<b>CDROM/DVD</b>"
#~ msgstr "<b>CD-ROM/DVD</b>"

#~ msgid "<b>Internet updates</b>"
#~ msgstr "<b>Hálózati frissítések</b>"

#~ msgid "<b>Internet</b>"
#~ msgstr "<b>Internet</b>"

#~ msgid ""
#~ "<i>To improve the user experience of Ubuntu please take part in the "
#~ "popularity contest. If you do so the list of installed software and how "
#~ "often it was used will be collected and sent anonymously to the Ubuntu "
#~ "project  on a weekly basis.\n"
#~ "\n"
#~ "The results are used to improve the support for popular applications and "
#~ "to rank applications in the search results.</i>"
#~ msgstr ""
#~ "<i>Az Ubuntu által biztosított felhasználói élmény javítása érdekében "
#~ "kérjük vegyen részt a népszerűségi versenyben. Ha így dönt, akkor a "
#~ "telepített szoftverek listája és azok használatának gyakorisága hetente "
#~ "összegyűjtésre és névtelenül elküldésre kerül az Ubuntu projektnek.\n"
#~ "\n"
#~ "Az eredmények a népszerű alkalmazások támogatásának javításához és a "
#~ "keresési eredmények rangsorolásához kerülnek felhasználásra.</i>"

#~ msgid "Add Cdrom"
#~ msgstr "_CD-ROM hozzáadása"

#~ msgid "Authentication"
#~ msgstr "Hitelesítés"

#~ msgid "D_elete downloaded software files:"
#~ msgstr "L_etöltött csomagok törlése:"

#~ msgid "Download from:"
#~ msgstr "Letöltés innen:"

#~ msgid "Import the public key from a trusted software provider"
#~ msgstr "Megbízható szoftverszolgáltató publikus kulcsának importálása"

#~ msgid "Internet Updates"
#~ msgstr "Hálózati frissítések"

#~ msgid ""
#~ "Only security updates from the official Ubuntu servers will be installed "
#~ "automatically"
#~ msgstr ""
#~ "Csak az Ubuntu kiszolgálóiról származó biztonsági frissítések kerülnek "
#~ "automatikusan telepítésre."

#~ msgid "Restore _Defaults"
#~ msgstr "_Alapértelmezés visszaállítása"

#~ msgid "Restore the default keys of your distribution"
#~ msgstr "A disztribúcióra jellemző alapértelmezett kulcsok visszaállítása"

#~ msgid "Software Sources"
#~ msgstr "Szoftverforrások"

#~ msgid "Source code"
#~ msgstr "Forráskód"

#~ msgid "Statistics"
#~ msgstr "Statisztika"

#~ msgid "Submit statistical information"
#~ msgstr "Statisztikai információk beküldése"

#~ msgid "Third Party"
#~ msgstr "Harmadik fél"

#~ msgid "_Check for updates automatically:"
#~ msgstr "_Frissítések automatikus keresése:"

#~ msgid "_Download updates automatically, but do not install them"
#~ msgstr "Frissítések automatikus _letöltése, a telepítésük nélkül"

#~ msgid "_Import Key File"
#~ msgstr "_Kulcsfájl importálása"

#~ msgid "_Install security updates without confirmation"
#~ msgstr "_Biztonsági frissítések telepítése megerősítés nélkül"

#~ msgid ""
#~ "<b><big>The information about available software is out-of-date</big></"
#~ "b>\n"
#~ "\n"
#~ "To install software and updates from newly added or changed sources, you "
#~ "have to reload the information about available software.\n"
#~ "\n"
#~ "You need a working internet connection to continue."
#~ msgstr ""
#~ "<b><big>Az elérhető szoftverekkel kapcsolatos információk elévültek</"
#~ "big></b>\n"
#~ "\n"
#~ "Szoftverek és frissítések telepítéséhez újonnan felvett vagy módosított "
#~ "forrásokból, újra le kell töltenie az elérhető szoftverekkel kapcsolatos "
#~ "információkat.\n"
#~ "\n"
#~ "A folytatáshoz működő hálózati kapcsolatra van szükség."

#~ msgid "<b>Comment:</b>"
#~ msgstr "<b>Megjegyzés:</b>"

#~ msgid "<b>Components:</b>"
#~ msgstr "<b>Összetevők:</b>"

#~ msgid "<b>Distribution:</b>"
#~ msgstr "<b>Disztribúció:</b>"

#~ msgid "<b>Type:</b>"
#~ msgstr "<b>Típus:</b>"

#~ msgid "<b>URI:</b>"
#~ msgstr "<b>URI:</b>"

#~ msgid ""
#~ "<big><b>Enter the complete APT line of the repository that you want to "
#~ "add as source</b></big>\n"
#~ "\n"
#~ "The APT line includes the type, location and components of a repository, "
#~ "for example <i>\"deb http://ftp.debian.org sarge main\"</i>."
#~ msgstr ""
#~ "<big><b>Adja meg a forrásként felvenni kívánt tároló teljes APT sorát</"
#~ "b></big>\n"
#~ "\n"
#~ "Az APT sor tartalmazza a tároló típusát, helyét és összetevőit, például "
#~ "<i>\"deb http://ftp.debian.org sarge main\"</i>."

#~ msgid "APT line:"
#~ msgstr "APT sor:"

#~ msgid ""
#~ "Binary\n"
#~ "Source"
#~ msgstr ""
#~ "Bináris\n"
#~ "Forrás"

#~ msgid "Edit Source"
#~ msgstr "Forrás szerkesztése"

#~ msgid "Scanning CD-ROM"
#~ msgstr "CD-ROM átvizsgálása"

#~ msgid "_Add Source"
#~ msgstr "_Forrás hozzáadása"

#~ msgid "_Reload"
#~ msgstr "F_rissítés"

#~ msgid "Configure the sources for installable software and updates"
#~ msgstr "Telepíthető szoftverek és frissítések forrásának beállítása"

#, fuzzy
#~ msgid "Software Sources Editor"
#~ msgstr "Szoftverforrások"

#~ msgid ""
#~ "http://changelogs.ubuntu.com/changelogs/pool/%s/%s/%s/%s_%s/changelog"
#~ msgstr ""
#~ "http://changelogs.ubuntu.com/changelogs/pool/%s/%s/%s/%s_%s/changelog"

#, fuzzy
#~ msgid "Ubuntu 7.04 'Feisty Fawn'"
#~ msgstr "Ubuntu 5.04 biztonsági frissítések"

#, fuzzy
#~ msgid "Cdrom with Ubuntu 7.04 'Feisty Fawn'"
#~ msgstr "Az Ubuntu 5.10 \"Breezy Badger\"-t tartalmazó CD-ROM"

#~ msgid "Ubuntu 6.10 'Edgy Eft'"
#~ msgstr "Ubuntu 6.10 \"Edgy Eft\""

#~ msgid "Community maintained"
#~ msgstr "Közösségi karbantartású"

#~ msgid "Proprietary drivers for devices"
#~ msgstr "Szabadalmazott eszközmeghajtók"

#~ msgid "Restricted software"
#~ msgstr "Nem-szabad"

#~ msgid "Cdrom with Ubuntu 6.10 'Edgy Eft'"
#~ msgstr "Az Ubuntu 6.10 \"Edgy Eft\" CD-ROM"

#~ msgid "Ubuntu 6.06 LTS 'Dapper Drake'"
#~ msgstr "Ubuntu 6.06 LTS \"Dapper Drake\""

#~ msgid "Canonical supported Open Source software"
#~ msgstr "A Canonical által támogatott nyílt forrású szoftverek"

#~ msgid "Community maintained (universe)"
#~ msgstr "Közösségi karbantartású (universe)"

#~ msgid "Community maintained Open Source software"
#~ msgstr "Közösségi karbantartású nyílt forrású szoftverek"

#~ msgid "Non-free drivers"
#~ msgstr "Nem-szabad meghajtók"

#~ msgid "Proprietary drivers for devices "
#~ msgstr "Szabadalmazott eszközmeghajtók "

#~ msgid "Restricted software (Multiverse)"
#~ msgstr "Nem-szabad szoftverek (Multiverse)"

#~ msgid "Software restricted by copyright or legal issues"
#~ msgstr "Szerzői vagy egyéb jogi problémák miatt korlátozott szoftver"

#~ msgid "Cdrom with Ubuntu 6.06 LTS 'Dapper Drake'"
#~ msgstr "Az Ubuntu 6.06 LTS \"Dapper Drake\"-et tartalmazó CD-ROM"

#~ msgid "Backported updates"
#~ msgstr "Visszaportolt frissítések"

#~ msgid "Ubuntu 5.10 'Breezy Badger'"
#~ msgstr "Ubuntu 5.10 'Breezy Badger'"

#~ msgid "Cdrom with Ubuntu 5.10 'Breezy Badger'"
#~ msgstr "Az Ubuntu 5.10 \"Breezy Badger\"-t tartalmazó CD-ROM"

#~ msgid "Ubuntu 5.10 Security Updates"
#~ msgstr "Ubuntu 5.10 biztonsági frissítések"

#~ msgid "Ubuntu 5.10 Updates"
#~ msgstr "Ubuntu 5.10 frissítések"

#~ msgid "Ubuntu 5.10 Backports"
#~ msgstr "Ubuntu 5.10 visszaportolt csomagok"

#~ msgid "Ubuntu 5.04 'Hoary Hedgehog'"
#~ msgstr "Ubuntu 5.04 \"Hoary Hedgehog\""

#~ msgid "Cdrom with Ubuntu 5.04 'Hoary Hedgehog'"
#~ msgstr "Az Ubuntu 5.04 \"Hoary Hedgehog\"-ot tartalmazó CD-ROM"

#~ msgid "Officially supported"
#~ msgstr "Hivatalosan támogatott"

#~ msgid "Ubuntu 5.04 Security Updates"
#~ msgstr "Ubuntu 5.04 biztonsági frissítések"

#~ msgid "Ubuntu 5.04 Updates"
#~ msgstr "Ubuntu 5.04 frissítések"

#~ msgid "Ubuntu 5.04 Backports"
#~ msgstr "Ubuntu 5.04 visszaportolt csomagok"

#~ msgid "Ubuntu 4.10 'Warty Warthog'"
#~ msgstr "Ubuntu 4.10 \"Warty Warthog\""

#~ msgid "Community maintained (Universe)"
#~ msgstr "Közösségi karbantartású (Universe)"

#~ msgid "Non-free (Multiverse)"
#~ msgstr "Nem-szabad (Multiverse)"

#~ msgid "Cdrom with Ubuntu 4.10 'Warty Warthog'"
#~ msgstr "Az Ubuntu 4.10 \"Warty Warthog\"-ot tartalmazó CD-ROM"

#~ msgid "No longer officially supported"
#~ msgstr "Hivatalosan már nem támogatott"

#~ msgid "Restricted copyright"
#~ msgstr "Szerzői jogi korlátozás alatt"

#~ msgid "Ubuntu 4.10 Security Updates"
#~ msgstr "Ubuntu 4.10 biztonsági frissítések"

#~ msgid "Ubuntu 4.10 Updates"
#~ msgstr "Ubuntu 4.10 frissítések"

#~ msgid "Ubuntu 4.10 Backports"
#~ msgstr "Ubuntu 4.10 visszaportolt csomagok"

#~ msgid "http://packages.debian.org/changelogs/pool/%s/%s/%s/%s_%s/changelog"
#~ msgstr "http://packages.debian.org/changelogs/pool/%s/%s/%s/%s_%s/changelog"

#~ msgid "Debian 3.1 \"Sarge\""
#~ msgstr "Debian 3.1 \"Sarge\""

#~ msgid "http://security.debian.org/"
#~ msgstr "http://security.debian.org/"

#~ msgid "Debian 3.1 \"Sarge\" Security Updates"
#~ msgstr "Debian 3.1 \"Sarge\" biztonsági frissítések"

#~ msgid "Debian \"Etch\" (testing)"
#~ msgstr "Debian \"Etch\" (tesztelés alatt)"

#~ msgid "http://http.us.debian.org/debian/"
#~ msgstr "http://http.us.debian.org/debian/"

#~ msgid "Debian \"Sid\" (unstable)"
#~ msgstr "Debian \"Sid\" (instabil)"

#~ msgid "DFSG-compatible Software with Non-Free Dependencies"
#~ msgstr "DFSG-kompatibilis szoftver nem-szabad függőségekkel"

#~ msgid "Non-DFSG-compatible Software"
#~ msgstr "Nem DFSG-kompatibilis szoftver"

#~ msgid "Downloading file %li of %li with unknown speed"
#~ msgstr "%li. fájl letöltése, összesen: %li, ismeretlen sebességgel"

#~ msgid "Normal updates"
#~ msgstr "Normális frissítések"

#~ msgid "Cancel _Download"
#~ msgstr "Letöltés _megszakítása"

#~ msgid "Some software no longer officially supported"
#~ msgstr "Néhány szoftver már nincs hivatalosan támogatva"

#~ msgid "Could not find any upgrades"
#~ msgstr "Nincs elérhető frissítés"

#~ msgid "Your system has already been upgraded."
#~ msgstr "Az Ön rendszere már frissítve lett."

#~ msgid ""
#~ "<span weight=\"bold\" size=\"x-large\">Upgrading to Ubuntu 6.10</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"x-large\">Frissítés az Ubuntu 6.10 "
#~ "változatra</span>"

#~ msgid "Important security updates of Ubuntu"
#~ msgstr "Az Ubuntu fontos biztonsági frissítései"

#~ msgid "Updates of Ubuntu"
#~ msgstr "Az Ubuntu frissítései"

#~ msgid "Cannot install all available updates"
#~ msgstr "Nem telepíthető minden rendelkezésre álló frissítés"

#~ msgid ""
#~ "<big><b>Examining your system</b></big>\n"
#~ "\n"
#~ "Software updates correct errors, eliminate security vulnerabilities and "
#~ "provide new features."
#~ msgstr ""
#~ "<big><b>A rendszer elemzése</b></big>\n"
#~ "\n"
#~ "A szoftverfrissítések programhibákat javítanak, megszüntetik a biztonsági "
#~ "sebezhetőségeket és új szolgáltatásokat biztosítanak."

#, fuzzy
#~ msgid "Oficially supported"
#~ msgstr "Hivatalosan támogatott"

#~ msgid ""
#~ "Some updates require the removal of further software. Use the function "
#~ "\"Mark All Upgrades\" of the package manager \"Synaptic\" or run \"sudo "
#~ "apt-get dist-upgrade\" in a terminal to update your system completely."
#~ msgstr ""
#~ "Néhány frissítés további szoftverek eltávolítását igényli. Használja a "
#~ "Synaptic csomagkezelő \"Minden frissítés kijelölése\" funkcióját, vagy "
#~ "futtassa terminálból a \"sudo apt-get dist-upgrade\" parancsot a rendszer "
#~ "teljes frissítéshez."

#~ msgid "The following updates will be skipped:"
#~ msgstr "A következő frissítések ki lesznek hagyva:"

#~ msgid "About %li seconds remaining"
#~ msgstr "Kb. %li másodperc van hátra"

#~ msgid "Download is complete"
#~ msgstr "A letöltés befejeződött"

#~ msgid "The upgrade aborts now. Please report this bug."
#~ msgstr "A frissítés félbeszakadt. Kérem, jelentse ezt hibaként."

#~ msgid "Upgrading Ubuntu"
#~ msgstr "Az Ubuntu frissítése"

#~ msgid "Hide details"
#~ msgstr "Részletek elrejtése"

#~ msgid "Show details"
#~ msgstr "Részletek megjelenítése"

#~ msgid "Only one software management tool is allowed to run at the same time"
#~ msgstr "Csak egy szoftverkezelő eszköz futhat egyidejűleg"

#~ msgid ""
#~ "Please close the other application e.g. 'aptitude' or 'Synaptic' first."
#~ msgstr ""
#~ "Zárja be a másik alkalmazást, például az aptitude vagy Synaptic "
#~ "programokat."

#~ msgid "<b>Channels</b>"
#~ msgstr "<b>Csatornák</b>"

#~ msgid "<b>Keys</b>"
#~ msgstr "<b>Kulcsok</b>"

#~ msgid "Installation Media"
#~ msgstr "Telepítő média"

#~ msgid "Software Preferences"
#~ msgstr "Szoftver beállításai"

#~ msgid "     "
#~ msgstr "     "

#~ msgid "<b>Channel</b>"
#~ msgstr "<b>Csatorna</b>"

#~ msgid "<b>Components</b>"
#~ msgstr "<b>Összetevők</b>"

#~ msgid "Add Channel"
#~ msgstr "Csatorna hozzáadása"

#~ msgid "_Add Channel"
#~ msgid_plural "_Add Channels"
#~ msgstr[0] "Csatorna hozzá_adása"

#~ msgid "_Custom"
#~ msgstr "_Egyéni"

#~ msgid "Ubuntu 6.06 LTS"
#~ msgstr "Ubuntu 6.06 LTS"

#~ msgid "Ubuntu 6.06 LTS Security Updates"
#~ msgstr "Ubuntu 6.06 LTS biztonsági frissítések"

#~ msgid "Ubuntu 6.06 LTS Updates"
#~ msgstr "Ubuntu 6.06 LTS frissítések"

#~ msgid "Ubuntu 6.06 LTS Backports"
#~ msgstr "Ubuntu 6.06 LTS visszaportolt csomagok"
